libsolidity/parsing/Parser.cpp

```diff
@@ -968,19 +968,46 @@ ASTPointer<UsingForDirective> Parser::parseUsingDirective()
 	expectToken(Token::Using);
 
 	vector<ASTPointer<IdentifierPath>> functions;
+	vector<optional<Token>> operators;
 	bool const usesBraces = m_scanner->currentToken() == Token::LBrace;
 	if (usesBraces)
 	{
 		do
 		{
 			advance();
 			functions.emplace_back(parseIdentifierPath());
+			if (m_scanner->currentToken() == Token::As)
+			{
+				advance();
+				Token operator_ = m_scanner->currentToken();
+				vector<Token> overridable = {
```

Comment:
This sounds like a boolean.
```suggestion
				vector<Token> const overridableOperators = {
```

Also, seems like it would be useful to have as a publicly exposed constant so that we can assert in various places that the token we got is actually usable as an operator (e.g. in `AST.h`).

Reply:
I renamed the variable, but I am not sure if is we want to make it a publicly exposed constant in the scope of this ticket.